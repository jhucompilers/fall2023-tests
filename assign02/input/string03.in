var s;
s = "a\nb\tc\n\"foo\"";
println(s);
44;
