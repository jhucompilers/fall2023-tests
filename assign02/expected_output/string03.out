a
b	c
"foo"
Result: 44
